app/assets/images/circle-question.svg

```diff
@@ -1,4 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="54px" height="54px">
-  <path fill="none" d="M0 0h24v24H0z"/>
-  <path fill="#ffffff" d="M12 22C6.477 22 2 17.523 2 12S6.477 2 12 2s10 4.477 10 10-4.477 10-10 10zm0-2a8 8 0 1 0 0-16 8 8 0 0 0 0 16zm-1-5h2v2h-2v-2zm2-1.645V14h-2v-1.5a1 1 0 0 1 1-1 1.5 1.5 0 1 0-1.471-1.794l-1.962-.393A3.501 3.501 0 1 1 13 13.355z"/>
-</svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="54" height="54">
```

Comment:
Looks like prettier ran when I merged master before committing.

Reply:
If Prettier ran, does this mean we want to roll this change back?

Reply:
Actually prettier doesn't run on svg, so not sure what happened here.

Reply:
probably SVGO.